### ISSUE TITLE
`tracef` does not format arguments when running using `w4 run-native`

### ISSUE THREAD

**Issue by TimDeve** [aduros/wasm4#417]
`tracef` does not format arguments when running using `run-native`. 

Example code:

```c
#include "wasm4.h"

void start() {
  tracef("String: %s", "Ok");
  tracef("Int: %d", 42);
}
```

Example output with `w4 run`:
```
String: Ok
Int: 42
```

Example out with `w4 run-native`:
```
String: %s
Int: %d
```

Edit: 
I wrote this issue before I found [the commit responsible](https://github.com/aduros/wasm4/commit/429ede2589a5bfda18c7b86e07f0d5850a462eb1), I feel it's still worthwhile to have an issue to track it if that's okay?

**Comment by aduros** [comment]
Yes, thanks for opening this!

I have no idea why enabling `vprintf` crashes on release builds only, but if you get it working, please open a PR :smile:

**Comment by JerwuQu** [comment]
This was fixed in #599 and can be closed.
